wiki/FAQ/en.md

```diff
@@ -56,161 +77,225 @@ _[Osu! Tatakae! Ouendan][ouendan wikipedia]_ is a rhythm game for the Nintendo D
+
+On the right of the health bar is your score. Below that is your accuracy. The circle beside your accuracy (and below your score) is a timer for the duration of the beatmap. The number on the bottom left is your combo counter/score multiplier.
 
 ---
 
 ## Alternative Game Modes
+### How to change game mode?
+![Mode change interface][Mode Interface image]
 
-### What is "osu!taiko"?
+_Image showcasing the Mode toolbox in Song Selection (Solo)_
+
+Click on the _Mode_ button to **open up a list of gameplay modes available on osu!**. Click on the desired gameplay mode, and osu! will switch to that gameplay mode style, along with relevant leaderboard change. Alternatively, you can press `Ctrl` and `1` (![osu! icon][osu! icon link] [osu!standard][osu! wikilink]) / `2` ( ![osu!taiko icon][osu!taiko icon link] [osu!taiko][osu!taiko wikilink]) / `3` (![osu!catch icon][osu!catch icon link] [osu!catch][osu!catch wikilink]) / `4` (![osu!mania icon][osu!mania icon link] [osu!mania][osu!mania wikilink]) on the keyboard to quickly switch to the desired gameplay mode.
+
+### What is osu!taiko?
 _Full explanation: [osu!taiko][osu!taiko wikilink]_
 
 ![osu!taiko Interface][osu!taiko Interface image]
 
-_[Taiko no Tatsujin][Taiko no Tatsujin wikipedia]_ is a Taiko Drum rhythm game made by Namco, that involves the simulation of playing the drums to the beat of the music. In osu!, beatmaps can be played in the styling of Taiko no Tatsujin through the use of special Taiko game mode.
+_[Taiko no Tatsujin][Taiko no Tatsujin wikipedia]_ is a taiko drum rhythm game made by Namco. This game involves the simulation of playing the taiko drum to the beat of the music. In osu!, beatmaps can be played in the styling of Taiko no Tatsujin through the use of osu!taiko game mode.
 
-Like basic osu!, there are three elements of gameplay.
+Like osu! game mode, there are three elements of gameplay.
 
 #### Basic explanation
+##### Playfield
+![Taiko playfield][Taiko playfield image]
+
+If you have played _Taiko no Tatsujin_ before, the only difference is that there is no chibi dancers at the bottom (must be storyboarded), the pass rate is _half_ of the healthbar (not immediately obvious in the default skin; please manually find and get a decent taiko skin), Kiai Time triggers Go-Go Time, gameplay gimmick elements like hit balloons or forked paths are not implemented (only the bare-bone basics), and background display difference (osu!taiko-specific beatmaps usually has its own custom background, leaving upper section empty; a video or static background image-only will take the lower portion only; if storyboard or storyboard+video, it will take over all the available screen space except the gameplay-important parts).
```

Comment:
There is 108 pages in the skinning subforum (https://osu.ppy.sh/forum/15) of varying completion rate.

Using deadbeat's Skin Compendium instead: https://osu.ppy.sh/forum/t/180864 .